leptos_macro/src/params.rs

```diff
@@ -13,7 +13,13 @@ pub fn params_impl(ast: &syn::DeriveInput) -> proc_macro::TokenStream {
             .named
             .iter()
             .map(|field| {
-				let field_name_string = &field.ident.as_ref().expect("expected named struct fields").to_string();
+				let field_name_string = &field
+                    .ident
+                    .as_ref()
+                    .expect("expected named struct fields")
+                    .to_string()
+                    .trim_start_matches("r#")
```

Comment:
This is also how serde does it: https://github.com/serde-rs/serde/blob/04ff3e8f950eb9862759781d85ec29d906917b70/serde_derive/src/internals/attr.rs#L134